### ISSUE TITLE
Make dragging the app window better.

### ISSUE THREAD

**Issue by agoogleuser** [xournalpp/xournalpp#3981]
**Is your feature request related to a problem? Please describe.**
When Choosing colour using the tablet (in right hand note taking Layout), I often mis-click and drag the window, and it's annoying.

**Describe the solution you'd like**
Make Xournal++ only dragable from the top bar like most of the other apps.

**Comment by rolandlo** [comment]
Are you using the stable version? The issue was fixed by #3907 since then.

**Comment by agoogleuser** [comment]
Yes, But It's not fixed with me on Fedora 35 up to date from the store.
